test-poc/examples/pom.xml

```diff
@@ -53,6 +53,12 @@
             <groupId>com.microsoft.sqlserver</groupId>
             <artifactId>mssql-jdbc</artifactId>
         </dependency>
+        <!-- Temporary dependency until we figure out how we want to support OAuth -->
```

Comment:
@vaceksimon can you please remove this dependency, I initially thought it is necessary, but it turned out it's not

Reply:
:white_check_mark:

Reply:
thank you :-)